.ko.yaml

```diff
@@ -0,0 +1,10 @@
+defaultLdflagdss:
```

Comment:
Not possible to set multiple ldflags in the command line https://ko.build/advanced/faq/#how-can-i-set-ldflags